README.md

```diff
@@ -145,13 +130,19 @@ $ docker-compose down --volumes
 
 #### FAQ
 
-##### I keep getting the error `standard_init_linux.go:211: exec user process caused "no such file or directory"` when I use docker-compose.
+##### I see the error `standard_init_linux.go:211: exec user process caused "no such file or directory"` when I use docker-compose.
 
 You probably have [CRLF line endings](https://en.wikipedia.org/wiki/CRLF).
 This causes the hashbang in the docker-entrypoint to be `/bin/sh\r` instead of `/bin/sh`.
 This is probably because you have `git.autocrlf` set to true,
 [set it to `input`](https://stackoverflow.com/questions/10418975) instead.
 
+##### I see the error `/opt/rustwide/cargo-home/bin/cargo: cannot execute binary file: Exec format error` when running builds.
+
+You are most likely not on a Linux platform. Currently, running builds is only supported on `x86_64-unknown-linux-gnu`.
```

Comment:
we could add what running builds is possible through the `web` container :) 

( I'm happy to add it myself after this)